src/scss/custom/components/_footer.scss

```diff
@@ -0,0 +1,164 @@
+footer {
```

Comment:
Agreed 100% that we need a separate class, though FWIW I like `mofo-footer` for clarity.